SU-SWS/stanford-jumpstart-deployer
# Patch webform for php 7.1

# READY FOR REVIEW

# Summary
Another PHP 7 fix.
RE: https://www.drupal.org/node/2811063

# Needed By (Date)
- Sometime, anytime, before release 4.15

# Urgency
- Not urgent.

# Steps to Test

* Build a JSL website without the patch
* Switch PHP to 7.1
* Navigate to any page and see PHP error
* Apply patch and see error go away.

# Affected Projects or Products
- Errything.

# Associated Issues and/or People
- RE: https://www.drupal.org/node/2811063
